client/pages/modal.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export default class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({ deckName: null });
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const { value } = event.target;
+    this.setState({ deckName: value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('/api/create-deck', req)
+      .then(res => {
+        res.json();
+        this.props.closeModal();
+      })
+      .catch(err => console.error(err));
+  }
+
+  render() {
+    if (!this.props.show) return null;
+    return (
+      <div className='modal-background'>
+        <div className="sticky-container">
+          <div className="sticky-outer">
+            <div className="sticky">
+              <svg width="0" height="0">
+                <defs>
+                  <clipPath id="stickyClip" clipPathUnits="objectBoundingBox">
+                    <path d="M 0 0 Q 0 0.69, 0.03 0.96 0.03 0.96, 1 0.96 Q 0.96 0.69, 0.96 0 0.96 0, 0 0"
+                      />
+                  </clipPath>
+                </defs>
+              </svg>
+              <div className="sticky-content">
+                <form className="new-deck-form" onSubmit={this.handleSubmit}>
+                  <a className="xmark-close" onClick={this.props.closeModal}>
+                    <i className="fa-solid fa-xmark" />
+                  </a>
+                  <h1 className="sticky-header">Create New Deck</h1>
+                  <input className="sticky-input" onChange={this.handleChange} type="text" placeholder="e.g. HTML, CSS, JavaScript"/>
+                  <button className="sticky-submit">Continue</button>
```

Comment:
Be sure the pass a `value` prop here.

Reply:
Added a `value` prop here to my `input`

![image](https://user-images.githubusercontent.com/109567499/200407534-c97849f8-774e-4d2f-8f84-9549b599659f.png)